tests/fileTests.py

```diff
@@ -18,6 +18,7 @@ class FileTests(BaseTestAPI):
     updateSubmissionId = None
     filesSubmitted = False
     submitFilesResponse = None
+    status_check_time = {"year":2016,"month":01,"day":02,"hour":03,"minute":04,"second":05}
```

Comment:
Is this line being used in the tests anywhere (couldn't find it, but looked pretty quickly)?